web/src/settings_components.ts

```diff
@@ -31,12 +48,21 @@ export function get_sorted_options_list(option_values_object) {
             }
             return 0;
         };
+    } else {
+        comparator = (x, y) => x.order - y.order;
```

Comment:
Currently all the objects that we pass to this function has `order` field, but in tests we test this case of not having `order` field. Do we want to keep this code for handling the cases without `order`?

Reply:
I guess we might as well; I think I can imagine one wanting to sort by key.

Reply:
Updated the code to keep `order` field as optional.